app/models/user.rb

```diff
@@ -86,6 +88,12 @@ def password_reset_expired?
     reset_sent_at < 2.hours.ago
   end
 
+  # Defines a proto-feed.
+  # See "Following users" for the full implementation.
+  def feed
+    microposts
```

Comment:
```suggestion
    microposts.newest
```
tạo ra scope mà ko dùng đến à